lib/rubernetes/auth/source.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+
+require 'rubernetes/auth/kube_config'
+require 'rubernetes/auth/service_account'
+require 'forwardable'
+
+module Rubernetes
+  module Auth
+    # This class is used to decide on which auth mechanism to use (service_account? vs. kube_config?).
+    # It helps to run Rubernetes in development/local enviornments where a serviceaccount is missing.
+    class Source
+      extend Forwardable
+      def_delegators :@authenticator, :api_endpoint, :ssl_options, :auth_options
+      attr_accessor :authenticator
+
```

Comment:
Since there's an `attr_accessor` for `@authenticator`, will `def_delegators :authenticator` work without the `@`?

Reply:
You are right. But I prefer to access the member `@authenticator` directly from within the class vs. utilizing the getter `authenticator`. + I do not have to worry about the lines ordering :)